lib/Mojo/WebSocketProxy/CallingEngine.pm

```diff
@@ -100,6 +99,8 @@ sub error_api_response {
     return $api_response;
 }
 
+my $request_no = 0;
```

Comment:
on restart it will reset $request_no to = 0, but as pid ($$) will be changed too, we'll get needed short-term uniquness.